single_cell/utils/bamutils.py

```diff
@@ -153,3 +153,52 @@ def bam_view(bam, output, region, **kwargs):
 
     pypeliner.commandline.execute(*cmd, **kwargs)
 
+
+def biobloom_categorizer(fastq1, fastq2, docker_image):
+    cmd = [
+        "biobloomcategorizer",
+        "--fq",
+        "-e",
+        "-p",
+        "/biobloom_output/biobloom",
+        "-f",
+        "/refdata/GCF_002021735.1_Okis_V1_genomic.bf /refdata/GRCh37-lite.bf /refdata/mm10_build38_mouse.bf",
+        fastq1,
+        fastq2,
+    ]
+    pypeliner.commandline.execute(*cmd, docker_image=docker_image)
+    delete_biobloom_output("/biobloom_output")
```

Comment:
dont need to delete these files if they're stored in temp dir... 

they'll get cleaned up automatically


just use the paths for alignment